Either/flipEither

```diff
@@ -0,0 +1,30 @@
+{-
+
+Flip the value of an Either where `Right` goes to `Left`, and `Left` goes to `Right`.
+
+Examples:
+```
+    let Either = ./Either/Type
+
+in  let E = constructors (Either Text Natural)
+
+in  ./Either/flipEither Text Natural (E.Left "ohno")
```

Comment:
Should probably just rename this to `flip`. Also, it’s probably type-classable, like Bifunctor-ish?

Reply:
Did a hoogle: https://hackage.haskell.org/package/invertible-0.2.0.5/docs/Control-Invertible-BiArrow.html#v:invert

Reply:
I've leave it for now though and just rename to `flip`

Reply:
Actually, `Either` isn't an instance. Need to keep looking :)

Reply:
`Braided` it would seem https://hackage.haskell.org/package/hs-functors-0.1.3.0/docs/src/Data.Bifunctor.Braided.html